### ISSUE TITLE
Needed exclusive touch settings.

### ISSUE THREAD

**Issue by arun-kumar0105** [CEWendel/SWTableViewCell#319]
I'm using this controller in my project,at some place on touching multiple buttons at a time my app crashes.I have used exclusive touch to avoid such crashes.It would be be great if you could provide such settings.

**Comment by vasilii-kasnitski** [comment]
Agree. I have a view controller and want to disable multiple touches. But exlusiveTouch does not affect it. Please, advice something.

**Comment by vasilii91** [comment]
Agree. I have a view controller and want to disable multiple touches. But exlusiveTouch does not affect it. Please, advice something.
